pyteal/ast/gtxn_test.py

```diff
@@ -2,19 +2,63 @@
 
 from .. import *
 
+teal1Options = CompileOptions(version=1)
```

Comment:
Testing failure with version=1 is a bit meaningless, since the compiler does not support TEAL v1

Reply:
@jasonpaulos Gotcha - I used TEAL v1 because I tested for presence of the minimum TEAL version check with op codes introduced in v2 (e.g. `Op.gtxn.min_version` = 2).

I started down the version check path because I realized there wasn't test coverage.  It's unclear to me if it's worth pursuing.  And/or if a better testing approach with _reasonable_ effort exists.  What do you think?

Reply:
@jasonpaulos Got it and makes sense.  I removed the test via https://github.com/michaeldiamant/pyteal/pull/1/commits/6d9cb3c45c467e04ad839abbb935890325f651d6.